db/version_set.h

```diff
@@ -655,7 +655,7 @@ class Version {
 
   uint64_t GetSstFilesSize();
 
-  MutableCFOptions GetMutableCFOptions() { return mutable_cf_options_; }
+  const MutableCFOptions& GetMutableCFOptions() { return mutable_cf_options_; }
```

Comment:
It may be worthwhile to look at other uses of the function and see whether there is any problem.

Reply:
it's used in one place and few unit tests only.